File: app/src/main/java/com/mecong/tenderalarm/model/MediaEntity.java
package com.mecong.tenderalarm.model;

import android.database.Cursor;

public class MediaEntity {
    private int id;
    private String header;
    private String uri;

    public MediaEntity(int id, String header, String uri) {
        this.id = id;
        this.header = header;
        this.uri = uri;
    }

    public static MediaEntity fromCursor(Cursor cursor) {
        return new MediaEntity(cursor.getInt(cursor.getColumnIndex("_id")),
                cursor.getString(cursor.getColumnIndex("title")),
                cursor.getString(cursor.getColumnIndex("uri")));
    }

    public int getId() {
        return id;
    }

    public String getHeader() {
        return header;
    }

    public String getUri() {
        return uri;
    }
}

File: app/src/main/java/com/mecong/tenderalarm/model/MediaEntity.kt
package com.mecong.tenderalarm.model

import android.database.Cursor

class MediaEntity(val id: Int, val header: String, val uri: String) {

  companion object {
    @JvmStatic
    fun fromCursor(cursor: Cursor): MediaEntity {
      return MediaEntity(
        cursor.getInt(cursor.getColumnIndexOrThrow("_id")),
        cursor.getString(cursor.getColumnIndexOrThrow("title")),
        cursor.getString(cursor.getColumnIndexOrThrow("uri"))
      )
    }
  }

}